name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/GenericJSONQuoteFeed.java

```diff
@@ -344,7 +344,16 @@ else if (object instanceof String s)
             return LocalDate.parse(object.toString(), formatter);
         }
         if (object instanceof String s)
-            return YahooHelper.fromISODate(s);
+        {
+            try
+            {
+                return parseDateTimestamp(Long.parseLong(s));
```

Comment:
In general I agree that we should try to parse the string also as a timestamp.
However, I propose to reverse this - because the "normal/default" behavior is that the string is actually an ISO date.